rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -1021,7 +1017,7 @@ def training_states_labels_and_entities(
         # Mapping of example state hash to list of positive labels associated with
         # the state. Note that each individual 'label' instance is a list of ints.
```

Comment:
In the comment it says "list of ints", but the type annotation says "List[List[Text]]".
```suggestion
        # the state. Note that each individual 'label' instance is a list of strings.
```